packs/feats/eternal-legend-dedication.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>As your renown as a fighter grows, your Calling comes to fruition. Almost nothing can keep you down. During your daily preparations, choose clumsy, drained, or enfeebled. For the remainder of the day, when you roll a success against an effect that would impose that condition on you, you get a critical success instead. In addition, anytime you gain the chosen condition, reduce its value by 1. You can change the chosen condition with a single action that has the concentrate trait.</p>"
+            "value": "<p>As your renown as a fighter grows, your Calling comes to fruition. Almost nothing can keep you down. During your daily preparations, choose @UUID[Compendium.pf2e.conditionitems.Item.Clumsy], @UUID[Compendium.pf2e.conditionitems.Item.Drained], or @UUID[Compendium.pf2e.conditionitems.Item.Enfeebled]. For the remainder of the day, when you roll a success against an effect that would impose that condition on you, you get a critical success instead. In addition, anytime you gain the chosen condition, reduce its value by 1. You can change the chosen condition with a single action that has the concentrate trait.</p>"
```

Comment:
I feel like this one having a toggle makes it seem like it would do something, maybe hold off until we actually have `inflicts:` roll options.

Reply:
The `AdjustDegreeOfSuccess` is predicated on the roll option. Should I keep that rule element? Predicated on just the future proofed roll options?

Reply:
Sure, will make it easier to find the feat later when we look for all `inflicts:` roll options in predicates

Reply:
Oh, actually, can't do *just* the `inflicts` roll options since you have to choose one of them. I'll just hold off entirely for now.